request_a_govuk_domain/request/utils.py

```diff
@@ -28,3 +30,26 @@ def organisations_list() -> list:
         for row in reader:
             data.append((row[0], (row[0])))
     return data
+
+
+def is_central_government_flow(registrant_type: str) -> bool:
```

Comment:
I think the use of flow here is ambiguous. There are more than just "central gov" and "not central gov" flows in the application, as answers to the purpose question lead to other flows. So I would use a different term, and would rename the function `is_central_government_registrant` for instance.

Reply:
Done, renamed as discussed in the call